submissions/morenko/memory-game/js/index.js

```diff
@@ -0,0 +1,136 @@
+const listCards = [
+    {
+        id: 1,
+        front: 'assets/front/1.jpg'
+    },
+    {
+        id: 2,
+        front: 'assets/front/2.jpg'
+    },
+    {
+        id: 3,
+        front: 'assets/front/3.jpg'
+    },
+    {
+        id: 4,
+        front: 'assets/front/4.jpg'
+    },
+    {
+        id: 5,
+        front: 'assets/front/5.jpg'
+    },
+    {
+        id: 6,
+        front: 'assets/front/6.jpg'
+    },
+    {
+        id: 7,
+        front: 'assets/front/7.jpg'
+    },
+    {
+        id: 8,
+        front: 'assets/front/8.jpg'
+    },
+];
+const copylistCards = listCards.slice();
+const cards = listCards.concat(copylistCards); 
+const shuffleСards = () => cards.sort(() => Math.random() - 0.5);
+
+const reset = document.querySelector(".reset");
+reset.addEventListener('click', function () {
+    reloadBoard();
+});
+
+const board = document.querySelector('.board');
+const createCards = function() {
+    let arrCards = '';
+    cards.forEach(({id, front}) => {
+        arrCards = arrCards + `
+        <div data-id="${id}" class="card closed">
+            <div class="back">
+                <img src="assets/img/js_back.png" alt="back">
+            </div>
+            <div class="front">
+                <img src="${front}" alt="front">
+            </div>
+        </div>`
+    })
+    board.innerHTML = arrCards;
+};
+
+let countCards = cards.length;
+const loadCards = function() {
```

Comment:
There are no `loading` in this function. Name is confusing :)

Reply:
Also, this function is not used anywhere except in `reloadBoard`.

Why do you need it?